contracts/service-registry/src/migrations/v_0_4.rs

```diff
@@ -35,7 +36,7 @@ pub fn migrate_services(
         .map(|(service_name, service)| {
             let service = Service {
                 name: service.name,
-                coordinator_contract: service.service_contract,
+                coordinator_contract: coordinator_contract.clone(),
```

Comment:
was the issue that the contract address was wrong/empty? I suggest renaming the function accordingly